src/ray/rpc/worker_client.h

```diff
@@ -0,0 +1,58 @@
+#ifndef RAY_RPC_WORKER_CLIENT_H
+#define RAY_RPC_WORKER_CLIENT_H
+
+#include <thread>
+
+#include <grpcpp/grpcpp.h>
+
+#include "ray/common/status.h"
+#include "ray/rpc/client_call.h"
+#include "ray/util/logging.h"
+#include "src/ray/protobuf/worker.grpc.pb.h"
+#include "src/ray/protobuf/worker.pb.h"
+
+namespace ray {
+namespace rpc {
+
+/// Client used for communicating with a remote worker server.
+class WorkerTaskClient {
+ public:
+  /// Constructor.
+  ///
+  /// \param[in] address Address of the worker server.
+  /// \param[in] port Port of the worker server.
+  /// \param[in] client_call_manager The `ClientCallManager` used for managing requests.
+  WorkerTaskClient(const std::string &address, const int port,
+                   ClientCallManager &client_call_manager)
+      : client_call_manager_(client_call_manager) {
+    std::shared_ptr<grpc::Channel> channel = grpc::CreateChannel(
+        address + ":" + std::to_string(port), grpc::InsecureChannelCredentials());
+    stub_ = WorkerTaskService::NewStub(channel);
+  };
+
+  /// Push a task.
```

Comment:
```suggestion
  /// Assign a task to the work.
```
`push` is kind of confusing.